great_expectations/experimental/metric_repository/cloud_data_store.py

```diff
@@ -11,11 +13,41 @@
 
 
 class CloudDataStore(DataStore[StorableTypes]):
+    def __init__(self, context: AbstractDataContext):
+        super().__init__(context=context)
+        config = self._context._cloud_config
+        self._session = create_session(access_token=config.access_token)
+        self._url = (
+            f"{config.base_url}/organizations/{config.organization_id}/metric-runs"
+        )
+
+    def _construct_json_payload(self, metric_run: MetricRun):
```

Comment:
Since metric_run is a pydantic model can we use the `dict()` method to create most of the payload?

Reply:
A lot of the data returned when calling dict() is not needed and some is not serializable. The uuid fields still need to be cast to str.